tree/treeplayer/test/dataframe/dataframe_cache.cxx

```diff
@@ -163,3 +211,62 @@ TEST(Cache, NonCopiable)
    EXPECT_EQ(0, ret)
       << "The static assert was not triggered even if caching of columns of a non copiable type was requested";
 }
+
+auto treeName = "t";
+auto fileName = "fileName.root";
+
+TEST(Cache, TakeCarrays)
```

Comment:
this test belongs in `dataframe_simple` more than here

Reply:
You have a point. There is a strong link in the implementation of these two functionalities, actually one depends on another, but stopping at the interface level, they are two very different things. I moved it.